opengrok-indexer/src/main/java/org/opengrok/indexer/configuration/Configuration.java

```diff
@@ -315,6 +316,16 @@
         ON, OFF, DIRBASED, UIONLY
     }
 
+    public enum MeterRegistryType {
```

Comment:
Is there much value to having this be `configuration.Configuration.MeterRegistryType` instead of just `configuration.MeterRegistryType`? (Same question for that other, old enum.)

Reply:
Having this directly underneath `configuration` is certainly better. It seems I need to listen to my inner voices more carefully - had the same thought when introducing the enum and brushed it under carpet in the process of doing other changes.